docs/validators/upgrades/v5.0-upgrade.md

```diff
@@ -1,6 +1,6 @@
 # Upgrade Guide v5.0
 
-This document provides guidelines for node operators upgrading from Regen Ledger v5.0 to v5.0. For a general overview of the upgrade process, please refer to [Upgrade Overview](README.md).
+This document provides guidelines for node operators upgrading from Regen Ledger v4.x to v5.0. For a general overview of the upgrade process, please refer to [Upgrade Overview](README.md).
```

Comment:
This should be v4.1 - nodes will have had to already performed the upgrade to v4.1.